### ISSUE TITLE
Apps names in the apps list are not Accessible for screen readers

### ISSUE THREAD

**Issue by makhlwf** [ChrisTitusTech/winutil#3343]
### ⚙️ Issue Checklist

- [x] I have read the guidelines.
- [x] I checked for duplicate issues.
- [x] I searched for existing discussions.
- [x] I checked for an existing pull request that addresses this issue.

### 📜 What part of Winutil are you having issues with?

apps installer

### 📝 Provide a clear and concise description of the issue.

In the latest update, the main screen of the app—specifically the installer screen—has become inaccessible to the NVDA screen reader. The list of apps is displayed as checkboxes, but the app names are not announced by NVDA. However, the buttons that follow each checkbox, such as "Install" or "Update," remain accessible.

### 🔄 Steps to reproduce the issue.

1 Turn on NVDA screen reader
2 run winutil Using the Powershell Command
3 Once it opens navigate within the menu using the tab key and tell you reach to the apps list
4 You'll hear the app's names list as unselected check boxes but you will never hear what is really the app name is.
